### PR TITLE
Add new API for get build parameters

### PR DESCRIPTION
**Pull request by rayakoren**
Before submitting a pull request, Please make sure the following:

## Description

Please explain the changes you made.

Link to design (if applicable)

### Checklist
- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Formating and linting run locally successfully
- [ ] All tests passing
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: mta/mta_services_test.go [hunk 1]
```diff
@@ -1405,6 +1405,34 @@ var _ = Describe("MtaServices", func() {
...
+			Ω(messages).Should(BeEmpty())
+		})
+
+		It("Get build parameters in a non existing mta.yaml file", func() {
+			mtaPath := getTestPath("result", "mta.yaml")
+			_, _, err := GetBuildParameters(mtaPath, nil)
+			Ω(err).Should(HaveOccurred())
+		})
+	})
+
```
Comment (tal-sapan) [inline mta/mta_services_test.go:1435]: Please add a test with mta extensions

File: mta/mta_services_test.go [hunk 2]
```diff
@@ -1426,6 +1426,28 @@ var _ = Describe("MtaServices", func() {
...
+				BeforeAll: []ProjectBuilder{myBuilder},
+				AfterAll:  []ProjectBuilder{otherBuilder},
+			}
+
+			mtaPath := getTestPath("mta.yaml")
+			extPath := getTestPath("mta.mtaext")
+
+			buildParameters, messages, err := GetBuildParameters(mtaPath, []string{extPath})
+			Ω(err).Should(Succeed())
+			Ω(messages).Should(BeNil())
```
Comment (tal-sapan) [inline mta/mta_services_test.go:1446]: You are checking the messages are empty afterwards, please remove one of them

File: mta/mta_services_test.go [hunk 3]
```diff
@@ -1426,6 +1426,28 @@ var _ = Describe("MtaServices", func() {
...
+			otherBuilder := ProjectBuilder{
+				Builder: "otherbuilder",
+			}
+			oBuildParameters := ProjectBuild{
+				BeforeAll: []ProjectBuilder{myBuilder},
+				AfterAll:  []ProjectBuilder{otherBuilder},
+			}
+
+			mtaPath := getTestPath("mta.yaml")
+			extPath := getTestPath("mta.mtaext")
```
Comment (tal-sapan) [inline mta/mta_services_test.go:1442]: Please add a comment that build parameters are invalid in the mtaext and we don't merge them

Comment (allaVolkov) [reply mta/mta_services_test.go:1442]: added

File: mta/mta_services_test.go (hunk 2 above)
Comment (allaVolkov) [reply mta/mta_services_test.go]: removed

**Review by tal-sapan — APPROVED** [review]

**Pull request merged**